src/open_inwoner/components/templates/components/Messages/Messages.html

```diff
@@ -2,8 +2,8 @@
 
 <section class="messages">
     <header class="messages__header">
-        <h4 class="utrecht-heading-4">{{ subject }}</h4>
-        <p class="p p--muted">{{ status }}</p>
+        <h3 class="utrecht-heading-4">{{ subject }}</h3>
```

Comment:
Do we need to switch to `h3` if we keep styling corresponding to `h4` elements?

Reply:
Actually we do. And it should even be an H2 tag instead, but styled like an h4 tag. So I will change that. 
This messages template really needs a rework. Too many things that do not function like a heading were given a heading tag.
Remember: the HTML hierarchy of the **tags** need to be correct. For accessibility you will need to often make headings style like as if they are headings of different higher/lower levels, as long as their HTML hierarchy remains correct, in order to allow screenreaders to read-out a correct main headings-menu in the correct order. Screenreaders respond to semantic HTML, not to CSS.